android/app/src/main/java/updated/mysterium/vpn/exceptions/TopupPreconditionFailedException.kt

```diff
@@ -0,0 +1,3 @@
+package updated.mysterium.vpn.exceptions
+
+class TopupPreconditionFailedException: Exception()
```

Comment:
`class TopupPreconditionFailedException(message: String) : Exception(message)`
Let's add message parametr and pass there error message from BE